mediaapi/routing/routing.go

```diff
@@ -73,9 +73,13 @@ func Setup(
 		if r := rateLimits.Limit(req); r != nil {
 			return *r
 		}
+		respondSize := cfg.MaxFileSizeBytes
+		if *cfg.MaxFileSizeBytes == 0 {
```

Comment:
Need to check not `nil` here also to avoid a potential panic.

Reply:
That should be done at startup, will add a comment.

Reply:
If it can't be `nil`, why is it a pointer type to begin with? :-) Let's simplify that.